src/mono/CMakeLists.txt

```diff
@@ -135,6 +163,11 @@ if(TARGET_SYSTEM_NAME STREQUAL "Darwin")
   set(RID_PREFIX "osx")
   set(CORETARGETS "-p:TargetsUnix=true -p:TargetsOSX=true")
   set(NETCORE_HOST_PLATFORM "macos")
+elseif(TARGET_SYSTEM_NAME STREQUAL "Emscripten")
+  set(TARGET_WASM 1)
+  set(RID_PREFIX "osx")
+  set(COREARCH "wasm32")
+  set(CORETARGETS "-p:TargetsUnix=true -p:TargetsOSX=true")
```

Comment:
we can remove RID_PREFIX/COREARCH/CORETARGETS, we shouldn't need to calculate these ourselves in dotnet/runtime.